wikimedia/wikipedia-ios
# Apple Pay - Form UI

**Phabricator:**
https://phabricator.wikimedia.org/T345846

### Notes
This PR contains the donate form UI for Apple Pay. It is dependent on https://github.com/wikimedia/wikipedia-ios/pull/4619.

The easier design review asks are done - we can do a followup PR later if we have time. I will get the payment sheet to present in the next PR.

### Test Steps
1. On fresh install, force load announcements. Do this by backgrounding once, then pulling to refresh on the Explore feed. This populates the configs we need to display the form.
2. Go to Settings > Donate. Confirm tapping button updates textfield. Confirm tapping transaction fee updates textfield. Confirm entering a small amount triggers a minimum amount error. Confirm entering a large amount triggers a maximum amount error (USD only). 
3. On device settings, change region and language to the Netherlands. Retest form. Confirm email opt-in checkbox appears. Confirm form still works as expected with a different currency.
4. Confirm form works okay with Voice Over.
